src/torchmetrics/image/lpip.py

```diff
@@ -78,10 +78,10 @@ class LearnedPerceptualImagePatchSimilarity(Metric):
         >>> _ = torch.manual_seed(123)
         >>> from torchmetrics.image.lpip import LearnedPerceptualImagePatchSimilarity
         >>> lpips = LearnedPerceptualImagePatchSimilarity(net_type='vgg')
-        >>> img1 = torch.rand(10, 3, 100, 100)
-        >>> img2 = torch.rand(10, 3, 100, 100)
+        >>> img1 = (torch.rand(10, 3, 100, 100) * 2) - 1
```

Comment:
lets add comment with an explanation of what the -1 is for :)

Reply:
Just added a comment